src/index.css

```diff
@@ -0,0 +1,28 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+img:hover {
+  animation: shake 0.5s;
+  animation-iteration-count: infinite;
+}
+
```

Comment:
- [ ] Kindly use the `Lato` and `Gill Sans` fonts for your application. These are the official fonts used in the original design. Kindly import and use them 💻